### PR TITLE
Notification Theming, Social Links menu, Region updates, Language Switcher

### PR DESCRIPTION
**Pull request by jordan-caldwell**
## Summary
This branch contains work from 11-6 to 11-11 that related to the following PL PR: https://github.com/State-of-Rhode-Island-eCMS/ecms_patternlab/pull/31. 

The work included here is:
- Create custom block for Notifications
- Defined Admin Links, Header: Inner, User settings regions
- Updated notification and promotion to use formatted text
- Notification config tweaks
- Added language switcher block
- Add source button to paragraph text

## Metadata
| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| `CHANGELOG` reflects changes? | yes
| Did you perform browser testing? | no
| Risk level |  Medium
| Relevant links | https://thinkoomph.jira.com/browse/RIG-130

### CODE REVIEW

File: ecms_base/modules/custom/ecms_blocks/src/Plugin/Block/SiteNotificationsBlock.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    $this->entityTypeManager = $entityTypeManager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build(): array {
+
+    // Query all notifications.
+    $query = \Drupal::entityQuery('node')
```
Comment (pfrilling) [inline ecms_base/modules/custom/ecms_blocks/src/Plugin/Block/SiteNotificationsBlock.php:79]: You should use your injected entity_type.manager instead.
```
$query = $this->entityTypeManager->getStorage('node')->getQuery();
$query->condition('type', 'notification')
      ->condition('status', 1)
      ->sort('field_notification_global', "DESC");
```

File: ecms_base/themes/custom/ecms/config/optional/block.block.languageswitcher.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  id: 'language_block:language_interface'
+  label: 'Language switcher'
+  provider: language
+  label_display: '0'
+visibility:
+  language:
+    id: language
+    langcodes:
+      en: en
+      es: es
```
Comment (pfrilling) [inline ecms_base/themes/custom/ecms/config/optional/block.block.languageswitcher.yml:24]: Should the language visibility be used here?

File: ecms_base/themes/custom/ecms/templates/blocks/block--menu-block--social-navigation.html.twig [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{#
+/**
+ * @file
+ * Theme override for a menu block.
```
Comment (pfrilling) [inline ecms_base/themes/custom/ecms/templates/blocks/block--menu-block--social-navigation.html.twig:4]: The @file comment should be a little more specific.

File: ecms_base/themes/custom/ecms/templates/blocks/block--sitenotifications.html.twig [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{#
+/**
+ * @file
+ * Theme override for a menu block.
```
Comment (pfrilling) [inline ecms_base/themes/custom/ecms/templates/blocks/block--sitenotifications.html.twig:4]: Same here. This isn't a menu block.

File: ecms_base/themes/custom/ecms/templates/navigation/links--language-block.html.twig [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{#
+/**
+ * @file
+ * Theme override for a set of links.
```
Comment (pfrilling) [inline ecms_base/themes/custom/ecms/templates/navigation/links--language-block.html.twig:4]: Maybe change to 'Theme override for the language links.'

File: ecms_base/themes/custom/ecms/templates/navigation/menu--social-navigation.html.twig [hunk 6]
```diff
@@ -0,0 +1,25 @@
+{#
+/**
+ * @file
+ * Theme override to display a social navigationmenu.
```
Comment (pfrilling) [inline ecms_base/themes/custom/ecms/templates/navigation/menu--social-navigation.html.twig:4]: Add a space between 'navigation menu'.

**Review by pfrilling — CHANGES_REQUESTED** [review]
Found a few nit picks. Otherwise, looks good.

File: ecms_base/themes/custom/ecms/config/optional/block.block.languageswitcher.yml (hunk 2 above)
Comment (jordan-caldwell) [reply ecms_base/themes/custom/ecms/config/optional/block.block.languageswitcher.yml:24]: This is temporary and will get overwritten once we land on a solution.

**Review by pfrilling — APPROVED** [review]
Looks good to me.

**Pull request merged**